CMakeLists.txt

```diff
@@ -81,8 +87,8 @@ project_options(
   ${ENABLE_COVERAGE}
   ${ENABLE_SANITIZER_ADDRESS}
   ${ENABLE_SANITIZER_UNDEFINED_BEHAVIOR}
-  # ${ENABLE_SANITIZER_THREAD}
-  # ${ENABLE_SANITIZER_MEMORY}
+  ${ENABLE_SANITIZER_THREAD}
+  ${ENABLE_SANITIZER_MEMORY}
```

Comment:
These sanitizers cannot be enabled with each other

Reply:
`check_sanitizers_support` only checks if the compiler supports that variable. Maybe we should handle that as well.